docs/reference/delete_fragment.rst

```diff
@@ -0,0 +1,8 @@
+Article's Fragment
+==================
+
+This feature moves the removed fragment inputs in another DOM and thereafter
+this allows the cancellations of removed fragments.
+After the deletion of a fragment, each deleted fragment item in the fragment list
+contains a reset button (to undo the deletion) and the fragment title. This action triggers undo of
+the deletion of the fragment so the moved fragment input's (and elements) are moving in their original place.
```

Comment:
the deletion of the fragment so the deleted fragment inputs (and elements) are moved to their original place.